prov/gni/include/gnix.h

```diff
@@ -74,6 +74,10 @@ extern "C" {
 #include "gnix_mr_cache.h"
 #include "gnix_mr_notifier.h"
 
+#if HAVE_UDREG
```

Comment:
Ignore that comment.  I thought we were conditionally compiling this based on a configure option.